### DISCUSSION TITLE
Text to Speech Spanish language

### DISCUSSION THREAD

**Discussion by ainteligencia** [deepgram/community#854] [General help] [answered]
question nova-2 works for Text to Speech
in Spanish language? where can i find the python example code?

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jkroll-deepgram** [comment, marked as answer]
Hi @ainteligencia , currently our text-to-speech only supports English. I'd encourage you to check back later in the year as we are planning to expand our supported languages! We post announcements to our [changelog](https://deepgram.com/changelog) so that is one helpful resource to reference.

**Reply by ainteligencia** [reply]
hello, does deepgram have text-to-speech in spanish?
